Essentials/src/com/earth2me/essentials/Console.java

```diff
@@ -39,39 +41,47 @@ public CommandSender getCommandSender() {
         return ess.getServer().getConsoleSender();
     }
 
-    @Override public String getName() {
+    @Override
+    public String getName() {
         return Console.NAME;
     }
 
-    @Override public String getDisplayName() {
-        return Console.NAME;
+    @Override
+    public String getDisplayName() {
+        return Console.DISPLAY_NAME;
     }
 
-    @Override public void sendMessage(String message) {
+    @Override
+    public void sendMessage(String message) {
         getCommandSender().sendMessage(message);
     }
 
-    @Override public boolean isReachable() {
+    @Override
+    public boolean isReachable() {
         return true;
```

Comment:
Why did you change that?

Reply:
Auto formatting. I've checked the projects, turns out mostly there's a line break after annotations. Since there's no clear code style I decided to keep it this way. If you mind, I can revert.

Reply:
Please turn off auto formatting and keep a minimal diff when working on Essentials PRs. Reverting it would be ideal. 😄

Reply:
Waiting on #3701 to rebase.

Reply:
no rush, PRs usually take a little while before merging anyways

happy oktoberfest still 😄